### ISSUE TITLE
Keyboard Enter when in a form with a <button type-"submit"> shows geocode results then form is submitted

### ISSUE THREAD

**Issue by eharrow** [perliedman/leaflet-control-geocoder#191]
I'm sure the behaviour up until recently was that enter, even when on a form with submit button and the map, did not submit the form but 1.5.5 seems to.  A work around is to use ctrl-enter or cmd-enter on a Mac.

**Comment by perliedman** [comment]
Hi, thanks for reporting this!

I'm guessing [this change](https://github.com/perliedman/leaflet-control-geocoder/commit/aeb575a85e197a754f77f98e978d165e96d07800#diff-cb2640e0bae61c7ddd5681b932941f66L286) is the reason.

I must confess that I've already forgotten why I did that change, but apparently preventing default there caused some issue in Firefox.

You could try to re-insert that `preventDefault` and see what happens in Firefox - maybe there's another workaround we can use instead.

**Comment by eharrow** [comment]
The issue is cross browser at least in chrome (mac) and safari.

**Comment by perliedman** [comment]
Yeah, no doubt. What I'm saying is that _having_ `preventDefault` there caused some problem in Firefox, unfortunately I forgot exactly what.

**Comment by chris68** [comment]
@perliedman I caught the key event in the map div. Solved my problem.

<div style="width: 300px; height: 300px;" id="picture-map-canvas" onkeydown="if (event.which == 13) {event.preventDefault();}"></div>

I also did some tests in Firefox. Works like charms. In my opinion it is ok (maybe even wise) to not prevent the bubbling for key up/key down. However, for enter/return I do not see the advantage.  Not even for Firefox. 

I would prevent it for enter again...
